src/diffusers/loaders/lora_pipeline.py

```diff
@@ -417,6 +420,17 @@ def load_lora_into_text_encoder(
                     else:
                         if is_peft_version("<", "0.9.0"):
                             lora_config_kwargs.pop("use_dora")
+
+                if "lora_bias" in lora_config_kwargs:
+                    if lora_config_kwargs["lora_bias"]:
+                        if is_peft_version("<=", "0.13.2"):
+                            raise ValueError(
+                                "You need `peft` 0.13.3 at least to use `bias` in LoRAs. Please upgrade your installation of `peft`."
```

Comment:
Any suggestions?

Reply:
I'd just not mention the technical details to avoid confusion:

"The loaded adapter is not compatible with the current version of PEFT, please install version ..."

Reply:
I think that is a bit vague, though. For example, when we detect `use_dora` is True on an incompatible `peft` version we log a better error:

https://github.com/huggingface/diffusers/blob/c96bfa5c80eca798d555a79a491043c311d0f608/src/diffusers/loaders/lora_pipeline.py#L415

So, I think in that context, this error could make sense too?

Reply:
If we want to keep that detail, let's at least change `bias` to `lora_bias` then? Otherwise, there can be confusion because there is already a `bias` parameter in PEFT LoRA which has been supported since the beginning (it refers to the base layer bias though).